devito/passes/clusters/aliases.py

```diff
@@ -418,10 +449,198 @@ def choose(exprs, aliases, selector):
     return chosen, others
 
 
-def process(cluster, chosen, aliases, max_par, sregistry, platform):
+def make_schedule(cluster, aliases, in_writeto, options):
```

Comment:
note for reviewers: this is _not_ an entirely new function -- this stems from the former `def iter` (now deleted, see below) though it's been changed a little